docs/src/3_minutes.md

```diff
@@ -0,0 +1,64 @@
+# 3 minutes to Temporian
+
+This is a _very_ quick introduction to how Temporian works. For a complete tour of its capabilities, please refer to the [User Guide](/user_guide).
+
+## Events and `EventSets`
+
+The most basic unit of data in Temporian is an **event**. An event consists of a timestamp and a set of feature values.
+
+Events are not handled individually. Instead, events are grouped together into an **[EventSet](/reference/temporian/implementation/numpy/data/event_set)**.
+
+`EventSet`s are the main data structure in Temporian, and represent **[multivariate time sequences](/user_guide/#what-is-temporal-data)**. Note that "multivariate" indicates that each event in the time sequence holds several feature values, and "sequence" indicates that the events are not necessarily sampled at a uniform rate (in which case we would call it a time "series").
+
+You can create an `EventSet` from a pandas DataFrame, NumPy arrays, CSV files, and more. Here is an example of an `EventSet` containing four events and three features:
+
+```python
+evset = tp.EventSet(
+	timestamps=["04-02-2023", "06-02-2023", "07-02-2023", "07-02-2023"],
+	features={
+        "feature_1": [0.5, 0.6, NaN, 0.9],
+        "feature_2": ["red", "blue", "red", "blue"],
+        "feature_3":  [10, -1, 5, 5],
+	},
+    index=["feature_2"],
+)
+```
+
+An `EventSet` can hold one or several time sequences, depending on what its **[index](/user_guide/#index-horizontal-and-vertical-operators)** is.
+
+If the `EventSet` has no index, it will hold a single time sequence, which means that all events will be considered part of the same group and will interact with each other when operators are applied to the `EventSet`.
+
+If the `EventSet` has one (or many) indexes, it will hold one time sequence for each unique value (or unique combination of values) of the indexes, the events will be grouped by their index value, and operators applied to the `EventSet` will be applied to each time sequence independently.
+
+## Graph, `Nodes` and Operators
+
+There are two big phases in any Temporian script: graph **definition** and **evaluation**. This is a common pattern in computing libraries, and it allows us to perform optimizations before the graph is evaluated, share Temporian programs across different platforms, and more.
+
+Processing operations are performed by **operators**. For example, the [`tp.simple_moving_average()`](/reference/temporian/core/operators/window/simple_moving_average) operator computes the [simple moving average](https://en.wikipedia.org/wiki/Moving_average) of each feature in an `EventSet`. You can find documentation for all available operators [here](/reference/temporian/core/operators/all_operators).
+
```

Comment:
I'd put the code example immediately after this paragraph "Processing operations...", followed by the next explanations.

Reply:
Also, the wording at the beginning is a bit redundant. Alternative:
`A graph is created by applying **operators** to its nodes`.